roles/lb/templates/haproxy.cfg.j2

```diff
@@ -120,3 +131,12 @@ backend lmtp_backend
     server node2  {{ node2_private_ip }}:32525 check
     server node3  {{ node3_private_ip }}:32525 check
     server master {{ master_private_ip }}:32525 check backup
+
+backend logstash_backend
+	mode http
+	balance roundrobin
+	option httpchk
+
+    server node2  {{ node2_private_ip }}:30544 check
+    server node3  {{ node3_private_ip }}:30544 check
```

Comment:
I think we must not load balance on ES this way (round robin) but have a failover strategy (if the normal node does not respond, then fallback to a secondary one).